imports/plugins/core/taxes/lib/collections/schemas/taxcodes.js

```diff
@@ -11,12 +11,21 @@ export const TaxCodes = new SimpleSchema({
     unique: true
   },
   shopId: {
+    type: String
+  },
+  taxCode: {
     type: String,
-    optional: true
+    label: "Tax Code"
+  },
+  taxCodeProvider: {
+    type: String,
+    label: "Tax Code Provider"
   },
   ssuta: {
```

Comment:
Assuming this is correct, but just want to make sure `ssuta` is what this should say